src/store/activities/actions.js

```diff
@@ -6,18 +6,18 @@ export const addActivity = activity => ({
   activity,
 });
 
-export const startAddActivity = (activity = {}) => dispatch => {
-  return database
-    .ref('activities/items')
-    .push(activity)
-    .then(ref => {
-      dispatch(
-        addActivity({
-          id: ref.key,
-          ...activity,
-        }),
-      );
-    });
+export const startAddActivity = (activity = {}) => async dispatch => {
+  try {
+    const id = await database.ref('activities/items').push(activity).key;
+    return dispatch(
+      addActivity({
+        id,
+        ...activity,
+      }),
+    );
+  } catch (error) {
+    return console.error(`Ops! ${error}`); // eslint-disable-line no-console
```

Comment:
console.logs are returning `undefined`, so there is not need for this return, as there are no code after it.

Reply:
eslint didn't allow to remove the return word :/ "9:67  error  Expected to return a value at the end of async arrow function  consistent-return"